pkg/controller/miganalytic/volume_adjustment.go

```diff
@@ -30,6 +30,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const DEFAULT_PV_USAGE_THRESHOLD = 3
```

Comment:
With this 3% threshold, is it possible that folks will end up with a new PV getting provisioned that is smaller than the one they had before?

Reply:
@djwhatle the users will get `max(93 gigs, 10 gigs, 100 gigs) = 100 gigs`. `93 gigs` comes from `90 gigs + 3%*100 gigs`. This `3%` becomes configureable with this PR.

Reply:
+1, that makes sense.